routers/root.py

```
@@ -242,16 +243,52 @@ def home_page(request: Request):
-    except composio_client.NotFoundError:
-        raise HTTPException(status_code=404)
-    return JSONResponse(content=tool.to_dict())
+        tool = INBUILT_TOOL_MAP[tool_slug](None, {})
+    except KeyError:
+        import composio_client
+        from composio import Composio
+
+        try:
+            tool = Composio().tools.get_raw_composio_tool_by_slug(slug=tool_slug)
+        except composio_client.NotFoundError:
+            raise HTTPException(status_code=404)
```

Ruff (0.16.0)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)